grounding.py

```diff
@@ -103,6 +104,6 @@ def ground(self, imagefile, segments, _=None):
         classes = list(classes)
 
         classes = classes_to_numpy(classes)
-        print "classified ", numpy.count_nonzero(classes != classes_to_numpy(BLANK_CLASS)), "characters out of", max(
-            classes.shape)
+        print(("classified ", numpy.count_nonzero(classes != classes_to_numpy(BLANK_CLASS)), "characters out of", max(
```

Comment:
Well, `2to3` isn't necessarily wrong, it just doesn't check for braces when it adds them, as it assumes only Python 2 code.